imap/httpd.c

```diff
@@ -2595,12 +2597,7 @@ EXPORTED const char *http_statusline(unsigned ver, long code)
 {
     static struct buf statline = BUF_INITIALIZER;
 
-    if (ver == VER_2) buf_setcstr(&statline, HTTP2_VERSION);
-    else {
-        buf_setmap(&statline, HTTP_VERSION, HTTP_VERSION_LEN-1);
-        buf_putc(&statline, ver + '0');
-    }
-
+    buf_setcstr(&statline, http_versions[ver]);
```

Comment:
... then you could add an assertion here for safety:

```suggestion
    assert(ver < n_http_versions);
    buf_setcstr(&statline, http_versions[ver]);
```

Reply:
done